orc8r/protos/magmad.proto

```diff
@@ -93,6 +93,25 @@ message LogLine {
   string line = 1;
 }
 
+message StatelessAgwMode {
+  enum AgwMode {
```

Comment:
Prefer making the zero value the invalid enum -- this way, in Go e.g., forgetting to initialize an enum value is clearly "invalid" rather than implicitly "stateless"